src/admin/content-block/components/ContentBlockForm/ContentBlockForm.tsx

```diff
@@ -128,10 +128,17 @@ const ContentBlockForm: FunctionComponent<ContentBlockFormProps> = ({
 			onToggle={setIsAccordionOpen}
 		>
 			{renderFormGroups(contentBlock.block.state.blockType, components, 'components')}
-			{renderFormGroups(contentBlock.block.state.blockType, block, 'block')}
 			{Array.isArray(components.state) && (
-				<Button icon="add" type="secondary" onClick={addComponentToState} />
+				<Spacer margin="bottom">
+					<Button
+						label={`Voeg ${get(contentBlock.components, 'name', '').toLowerCase()} toe`}
```

Comment:
missing translation
make sure to follow: https://studiohyperdrive.slack.com/archives/CJ55B3Q2W/p1578467065013900